src/Tools/BuildBoss/.gitignore

```diff
@@ -0,0 +1 @@
+launchSettings.json
```

Comment:
I would prefer to see this as the following line in the main file. That way it's less likely to stay in the source tree forever.

```
/src/Tools/BuildBoss/Properties/launchSettings.json
```